File: backend/src/main/java/com/lawencon/ticketms/dto/ticket/TicketUpdateResDto.java
package com.lawencon.ticketms.dto.ticket;

public class TicketUpdateResDto {
	private TicketUpdateDataResDto data;
	private String message;
	public TicketUpdateDataResDto getData() {
		return data;
	}
	public void setData(TicketUpdateDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/ticket/ticket.update.res.dto.ts
import { TicketUpdateDataResDto } from "./ticket.update.data.res.dto";

export class TicketUpdateResDto{
    private _data! : TicketUpdateDataResDto
	private _message! : string

    public get data(): TicketUpdateDataResDto {
        return this._data;
    }

    public set data(data: TicketUpdateDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }
    
}